backend/src/auth/auth.controller.ts

```diff
@@ -7,15 +7,18 @@ import { UserDocument } from 'src/users/schemas/user.schema';
 import { JwtAuthGuard } from './guards/jwt-auth.guard';
 import { Request, Response } from 'express';
 import { LoginDto } from './dto/login-auth.dto';
-import { ApiBody, ApiOperation, ApiResponse } from '@nestjs/swagger';
-
+import { ApiBody, ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { AuthenticatedUser } from './interfaces/authenticat-user.interface';
```

Comment:
There's a typo in the name of the interface file here if you want to update this

Reply:
I will work on it and address the concerns thank you.